services/common/src/components/projectSummary/AuthorizationsInvolved.tsx

```diff
@@ -177,7 +182,8 @@ const RenderEMANewPermitSection = ({
   project_summary_guid,
 }) => {
   const new_props = {
-    ...props.formValues.authorizations.AIR_EMISSIONS_DISCHARGE_PERMIT.AMENDMENT[0],
+    ...props.formValues.authorizations[code],
```

Comment:
Should this change include the `AMENDMENT[0]` as it did before?

```suggestion
    ...props.formValues.authorizations[code].AMENDMENT[0],
```